### PR TITLE
Create 2024-03-25-global-ai-lesson-sprint.md

### PR DESCRIPTION
**Pull request by jt14den**
Report back from last month's lesson sprint for the  Intro to AI for GLAM lesson: https://carpentries-incubator.github.io/machine-learning-librarians-archivists/

### CODE REVIEW

File: _posts/2024/03/2024-03-25-global-ai-lesson-sprint.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+teaser: "Discover how a global code sprint is driving the evolution of AI education in libraries and shaping the future of the GLAM community."
+ title: "Advancing AI Education in GLAM: Highlights from the Library Carpentry Code Sprint"
+date: 2024-03-25
+time: "09:00:00"
+tags: ["Curriculum", "Library Carpentry", "Maintainers", "Curriculum Development", "Pilot Workshops"]
+---
+
+## Advancing AI Education in GLAM: Highlights from the Library Carpentry Code Sprint
+
+The lesson maintainers for [Intro to AI for GLAM (Galleries, Libraries, Archives, Museums)](https://carpentries-incubator.github.io/machine-learning-librarians-archivists/) and the [Library Carpentry Curriculum Advisory ](https://github.com/LibraryCarpentry/curriculum-advisors)Committee co-hosted a seven-hour online code sprint on Jan 31, 2024 to help contribute bug fixes, edits, and improvements to this popular new lesson.
```
Comment (tobyhodges) [inline _posts/2024/03/2024-03-25-global-ai-lesson-sprint.md]:
```suggestion
The lesson maintainers for [Intro to AI for GLAM (Galleries, Libraries, Archives, Museums)](https://carpentries-incubator.github.io/machine-learning-librarians-archivists/) and the [Library Carpentry Curriculum Advisory Committee](https://github.com/LibraryCarpentry/curriculum-advisors) co-hosted a seven-hour online code sprint on Jan 31, 2024 to help contribute bug fixes, edits, and improvements to this popular new lesson.
```

**Review by tobyhodges — APPROVED** [review]
Looks great to me. Thanks for taking the time to write this, @jt14den, @leighphan & @chennesy

**Comment by tobyhodges** [comment]
@OscarSiba please let us know if you'd prefer a different publication date for this. It is currently drafted for 25 March.

**Comment by OscarSiba** [comment]
> Advancing AI Education in GLAM: Highlights from the Library Carpentry Code Sprint

Hi @tobyhodges @jt14den the 25th is fine. However, it looks like not all checks were successful to deploy preview.

**Comment by tobyhodges** [comment]
> However, it looks like not all checks were successful to deploy preview.

Good catch, @OscarSiba. Fixed now!

**Pull request merged**